certora/harnesses/GuardianStorageHarness.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.8.12 <0.9.0;
+import {GuardianStorage} from "../../contracts/modules/social_recovery/storage/GuardianStorage.sol";
+
+contract GuardianStorageHarness is GuardianStorage {
+
+    /**
+     * @dev Gets the count of guaridans for a wallet.
+     * @param _wallet The target wallet.
+     * @return count of guardians.
+     */
+    function getGuardiansCount(address _wallet) public view returns (uint256 count) {
```

Comment:
nit: this name implies that it is reading the `guardianCount` storage but it isn't, maybe:

```suggestion
    function countGuardians(address _wallet) public view returns (uint256 count) {
```